samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/DocumentMemorySkill.cs

```diff
@@ -79,15 +79,13 @@ await foreach (var memory in results)
         foreach (var memory in relevantMemories)
         {
             var tokenCount = Utilities.TokenCount(memory.Metadata.Text);
-            if (remainingToken - tokenCount > 0)
-            {
-                documentsText += $"\n\nSnippet from {memory.Metadata.Description}: {memory.Metadata.Text}";
-                remainingToken -= tokenCount;
-            }
-            else
+            if (remainingToken - tokenCount <= 0)
             {
                 break;
```

Comment:
Kind of, Technically the way we store documents we break them into multiple memories if they are big. We don't do partial memory loads. So we could perhaps do partial document loads if the documents are very big but if they are small we either load them or we omit them.
I can add a warning around token limits.

Reply:
It will not include any more document snippets if it doesn't have enough remaining tokens. It won't have a partial snippet. The snippets have been sorted by relevancy and the token count for each snippet should be small enough such that the most relevant snippets should be included in the prompt. 

I agree that we should probably throw a warning, same for semantic memory.